cluster-up/cluster/kind/common.sh

```diff
@@ -201,9 +201,9 @@ function _kubectl() {
 
 function down() {
     _fetch_kind
-    if [ -z $($KIND get clusters | grep ${CLUSTER_NAME}) ]; then
+    if [ -z "$($KIND get clusters | grep ${CLUSTER_NAME})" ]; then
```

Comment:
nit:
the PR fixes those as well in case you want to write in the the description,
not mandatory, as the main reason is the ipv6 of course.

question
do we want to start adding the git log of the new commits for each bump ?
as part of the commit message

Reply:
Unless it is automated it would be too much hassle. And I don't see a huge reason to. We are not including logs of all our Go dependencies when we bump then neither, for obvious reasons.

Reply:
we can automate it easily if needed